Wrappers/Python/cil/recon/FBP.py

```diff
@@ -450,12 +492,11 @@ def set_split_processing(self, slices_per_chunk=0):
     def _calculate_weights(self, acquisition_geometry):
 
         ag = acquisition_geometry
-        weight = 0.5 * np.pi/ ag.num_projections
-        if self.backend=='tigre':
-            weight /= ag.pixel_size_h
-        else:
-            weight /= ag.pixel_size_h * ag.pixel_size_v
-        self._weights = np.full((ag.pixel_num_v,ag.pixel_num_h),weight,dtype=np.float32)
+        scaling = 0.25 * (2 * np.pi/ ag.num_projections) / ag.pixel_size_h
+
+        if self.backend=='astra':
+            scaling /=  ag.pixel_size_v
+        self._weights = np.full((ag.pixel_num_v,ag.pixel_num_h),scaling,dtype=np.float32)
```

Comment:
Nothing is really changed here?

Reply:
Nope, same maths. Just cleaned up slightly.